frontend/src/app/shared/services/countdown.service.ts

```diff
@@ -16,25 +17,24 @@ export class CountdownService {
       const timerId =
         countdown(
           new Date(now.getFullYear(), now.getMonth(), now.getDate(), hour, minute, 0),
-          ts => {
-            // if minutes or seconds are < 10 add a 0 before
-            const time = `${('0' + ts.minutes).slice(-2)}:${('0' + ts.seconds).slice(-2)}`;
+          (ts: { hours: number, minutes: number, seconds: number }) => {
+            const time = getFormatedTime(ts);
 
             // tell listeners that a new value is available
             if (previousSecond && previousSecond > ts.seconds) {
               observer.next(time);
             }
 
             // stop the countdown if 0:0 or if already finished
-            if ((ts.minutes === 0 && ts.seconds === 0) ||  previousSecond && previousSecond < ts.seconds) {
+            if ((ts.minutes === 0 && ts.seconds === 0) || previousSecond && previousSecond < ts.seconds) {
```

Comment:
I think that if you have for example `02:01:00` left, at `02:00:00` the countdown will stop